tests/tck/features/bugfix/SubgraphBeforePipe.feature

```diff
@@ -107,14 +107,14 @@ Feature: Test get subgraph before pipe
       |                                                 |                       | [:serve "Boris Diaw"->"Jazz"@0]                  |
       |                                                 |                       | [:serve "Boris Diaw"->"Spurs"@0]                 |
       |                                                 |                       | [:serve "Boris Diaw"->"Suns"@0]                  |
-      |                                                 |                       | [:like "Yao Ming"->"Shaquile O\'Neal"@0]         |
-      |                                                 |                       | [:like "Shaquile O\'Neal"->"JaVale McGee"@0]     |
-      |                                                 |                       | [:serve "Shaquile O\'Neal"->"Cavaliers"@0]       |
-      |                                                 |                       | [:serve "Shaquile O\'Neal"->"Celtics"@0]         |
-      |                                                 |                       | [:serve "Shaquile O\'Neal"->"Heat"@0]            |
-      |                                                 |                       | [:serve "Shaquile O\'Neal"->"Lakers"@0]          |
-      |                                                 |                       | [:serve "Shaquile O\'Neal"->"Magic"@0]           |
-      |                                                 |                       | [:serve "Shaquile O\'Neal"->"Suns"@0]            |
```

Comment:
These blobs changed a lot, I'm hoping I got them right, but as noted earlier, I've left the whitespace adjustments as a distinct commit.